pkgs/applications/networking/instant-messengers/wavebox/default.nix

```diff
@@ -18,7 +18,7 @@
 }:
 
 let
-  version = "10.114.26-2";
+  version = "10.117.21-2";
```

Comment:
this attr can be moved out of the let once the unneeded tarball attr is removed.